Developer Guide.md

```diff
@@ -46,13 +46,41 @@ nodemon --watch dist -e js bin/xud --lndbtc.disable=true --lndltc.disable=true
 
 We recommend to connect the `xud` instance you are developing on with the `xud-simnet`. Combined with above restarting mechanism, this let's you instantly see how your changes behave in a real-world trading environment.
 
-1.
-2.
-3.
+1. Install xud-simnet by following [here](https://github.com/ExchangeUnion/xud/wiki/Docker)
```

Comment:
Install the `xud-simnet` as described [here](https://docs.exchangeunion.com/v/gitbook/start-trading/user-guide).